tough/src/sign.rs

```diff
@@ -27,6 +28,21 @@ pub trait Sign: Sync + Send {
     ) -> std::result::Result<Vec<u8>, Box<dyn std::error::Error + Send + Sync + 'static>>;
 }
 
+/// Implements `Sign` for a reference to any type that implements `Sign`.
+impl<'a, T: Sign> Sign for &'a T {
+    fn tuf_key(&self) -> Key {
+        (*self).tuf_key()
+    }
+
+    fn sign(
+        &self,
+        msg: &[u8],
+        rng: &dyn SecureRandom,
+    ) -> std::prelude::rust_2015::Result<Vec<u8>, Box<dyn Error + Send + Sync + 'static>> {
```

Comment:
```suggestion
    ) -> std::result::Result<Vec<u8>, Box<dyn Error + Send + Sync + 'static>> {
```

what happened here 😅

Reply:
Weird. Maybe fmt did that

Reply:
:smile: